Mobile/ShopLens/Droid/Listeners/CaptureSessionCallback.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Android.Hardware.Camera2;
+
+namespace ShopLens.Droid.Listeners
+{
+    public class CaptureSessionCallback : CameraCaptureSession.StateCallback
+    {
+        private readonly Camera2Fragment owner;
+
+        public CaptureSessionCallback(Camera2Fragment owner)
+        {
+            this.owner = owner ?? throw new System.ArgumentNullException("owner");
```

Comment:
Maybe this is descriptive enough, because it tells you the line at which you get it but still.